openfecli/tests/commands/test_gather.py

```diff
@@ -28,14 +28,25 @@ def test_get_column(val, col):
 
 
 @pytest.fixture
-def results_dir(tmpdir):
+def results_dir_serial(tmpdir):
+    """Example output data, with replicates run in serial (3 replicates per results JSON)."""
     with tmpdir.as_cwd():
         with resources.files('openfecli.tests.data') as d:
             t = tarfile.open(d / 'rbfe_results.tar.gz', mode='r')
             t.extractall('.')
 
         yield
 
+@pytest.fixture
+def results_dir_parallel(tmpdir):
```

Comment:
I don't actually use this fixture in this PR, since I just want to add the test data here and not add any tests that would fail.